docs/metrics/wer-cer-mer.md

```diff
@@ -66,14 +70,20 @@ $$
 $$
 
 ### Example
+
 Let's calculate the word error rate between the following reference and candidate texts:
 
+<!-- markdownlint-disable MD013 -->
```

Comment:
For my knowledge, why is this needed when we have `table` disabled in MD013 configuration?